pkgs/tools/misc/autorandr/default.nix

```diff
@@ -23,7 +26,12 @@ stdenv.mkDerivation rec {
     runHook preInstall
     make install TARGETS='autorandr' PREFIX=$out
 
-    make install TARGETS='bash_completion' DESTDIR=$out/share/bash-completion/completions
```

Comment:
I'm not sure if this is doing what's intended. Looking at the makefile I think this works by generating a path like ~`$out/share/bash-completion/completions//autorandr`

From the makefile https://github.com/phillipberndt/autorandr/blob/1.11/Makefile#L55
```
install -D -m 644 contrib/bash_completion/autorandr ${DESTDIR}/${BASH_COMPLETIONS_DIR}/autorandr
```